src/blockfilter.cpp

```diff
@@ -47,7 +47,7 @@ GCSFilter::GCSFilter(const Params& params)
     : m_params(params), m_N(0), m_F(0), m_encoded{0}
 {}
 
-GCSFilter::GCSFilter(const Params& params, std::vector<unsigned char> encoded_filter)
+GCSFilter::GCSFilter(const Params& params, std::vector<unsigned char> encoded_filter, bool filter_checked)
```

Comment:
In commit "Make sanity check in GCSFilter constructor optional" (c35df324b19b94aaecfbcd26111b7d00c5bed4e0)

Requesting change: Can we rename `filter_checked` to `skip_checks` or `skip_decode_check`. I spent an embarrassing amount of time being confused by the changes in this PR because I interpreted `filter_checked` to mean "filter should be checked" not "filter should not be checked" which is what it seems to actually mean.

Reply:
Changed `filter_checked` to `skip_decode_check`